src/main/java/com/powsybl/openloadflow/network/impl/AbstractLfGenerator.java

```diff
@@ -301,40 +301,47 @@ public void setParticipating(boolean participating) {
     }
 
     public static boolean checkActivePowerControl(String generatorId, double targetP, double minP, double maxP, double plausibleActivePowerLimit,
-                                                  LfNetworkLoadingReport report) {
+                                                  boolean useActiveLimits, LfNetworkLoadingReport report) {
         boolean participating = true;
         if (Math.abs(targetP) < POWER_EPSILON_SI) {
-            LOGGER.trace("Discard generator '{}' from active power control because targetP ({}) equals 0",
+            LOGGER.trace("Discard generator '{}' from active power control because targetP ({} MW) equals 0",
                     generatorId, targetP);
             if (report != null) {
                 report.generatorsDiscardedFromActivePowerControlBecauseTargetEqualsToZero++;
             }
             participating = false;
         }
+        if (maxP > plausibleActivePowerLimit) {
```

Comment:
I understand your point of view and why you prefer to keep this check even if limits are disabled but... it makes the feature less comprehensive.

Reply:
I think we need this check otherwise some active power distribution mode based on pmax won't work anymore. An alternative would be to forbid these active power distribution modes in case of useActiveLimits to false but this was not the idea of this feature.

Reply:
@jeandemanged why do you say "factors for fictitious elements."? Ficticious?

Reply:
Some TSOs provide provide generators with very high Pmax in their IGMs, equivalents would be a better term instead of fictitous